src/main/java/uk/gov/ons/ctp/integration/mockcaseapiservice/endpoint/CaseServiceMockStub.java

```diff
@@ -250,6 +250,23 @@ private void nullTestThrowsException(Object response) {
     }
   }
 
+  /**
+   * Post a list of Cases in order to modify those in the case maps driving the responses here.
+   *
+   * @param requestBody - a list of cases
+   * @return - response confirming post.
+   */
+  @RequestMapping(value = "/data/cases/modify", method = RequestMethod.POST)
+  @ResponseStatus(value = HttpStatus.OK)
+  public ResponseEntity<ResponseDTO> modifyCaseData(@RequestBody List<CaseContainerDTO> requestBody)
+      throws CTPException {
+
+    log.with("requestBody", requestBody).info("Entering POST modifyCData");
```

Comment:
Is that a typo? I'd just put  "modify Case Data" as it's unambiguous. If it was midnight and I was reading the log message I'd hit the vodka....